exporter.go

```diff
@@ -49,28 +53,33 @@ func (e *Exporter) ExportSpans(ctx context.Context, spans []sdktrace.ReadOnlySpa
 		return nil
 	}
 
-	var lumigoSpans []telemetry.Span
 	e.encoderMu.Lock()
 	defer e.encoderMu.Unlock()
+
 	for _, span := range spans {
 		mapper := transform.NewMapper(e.context, span, logger)
 		lumigoSpan := mapper.Transform()
+		e.spansTotalSizeBytes += int(reflect.TypeOf(lumigoSpan).Size())
```

Comment:
This means that the value will keep growing even if no span is added, maybe it's better to save it to a local variable first and assign it to `e.spansTotalSizeBytes` only later?